.github/workflows/_runner-determinator.yml

```diff
@@ -98,26 +107,28 @@ jobs:
               return branch.split("/")[0] in {"main", "nightly", "release", "landchecks"}
 
 
-          def get_workflow_type(issue: Issue, username: str) -> Tuple[str, str]:
-              formatted_username = (
-                  f"@{username}"  # Add the @ symbol to match the format in the issue body
-              )
-
+          def get_workflow_type(issue: Issue, username_check: Iterable[str]) -> Tuple[str, str]:
               try:
-                  user_list = issue.get_comments()[0].body.split()
+                  first_comment = issue.get_comments()[0].body.strip("\n\t ")
```

Comment:
nit: `first_comment` doesn't really explain the semantics of what data is stored in the variable

Reply:
isn't it the body of the first comment? Just stripped from any trailing and leading white space char?